### ISSUE TITLE
EGLL DET SID

### ISSUE THREAD

**Issue by jimihendrix272** [flybywiresim/a32nx#7231]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2022-05-22T15:30:00+00:00",
    "ref": "refs/heads/master",
    "sha": "d52a9f331e49ac9f91a6e71334d849accba9f035",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:d52a9f33"
}
```

### Describe the bug

At EGLL, on 27R and 27L DET departures do not follow the turn towards DET. 
It should be a (sharp) left turn but it turns right instead.
I control at Heathrow and quite often other aircraft also incorrectly turn right in the DET deparatures. :)

### Expected behavior

As above

### Steps to reproduce

Every time

### Discord Username (optional)

jimihendrix#7907

**Comment by tracernz** [comment]
Is it drawn incorrectly, or it just turns incorrectly?

**Comment by jimihendrix272** [comment]
It is drawn correctly (not exactly the same as the SID chart, but close enough), but on autopilot, instead of a left turn, it insists on turning to the right.
Today I tried the departure and as soon as it started turning right I used heading mode to turn it left, but even after getting back on the proper flight path and selecting NAV mode again it seemed to want to turn to the right for a while.
Thanks for your reply.
John Cardiff
